core/pkg/registryadaptors/GCP/v1/gcpadaptorutils.go

```diff
@@ -0,0 +1,48 @@
+package v1
+
+import (
+	"fmt"
+
+	"github.com/kubescape/kubescape/v2/core/pkg/registryadaptors/registryvulnerabilities"
+	grafeaspb "google.golang.org/genproto/googleapis/grafeas/v1"
+)
+
+func (GCPAdaptor *GCPAdaptor) getImageLastScanId(imageID *registryvulnerabilities.ContainerImageIdentifier) (string, error) {
+	var s string
+
+	return s, nil
+}
+
+func GetResourceUrl(imageID *registryvulnerabilities.ContainerImageIdentifier) string {
+	return fmt.Sprintf("https:/%s/%s", imageID.Repository, imageID.Tag)
```

Comment:
I think you might have missed a `/` here? 

```suggestion
	return fmt.Sprintf("https://%s/%s", imageID.Repository, imageID.Tag)
```

Reply:
Use the `url` package for proper generating URL strings

Reply:
oh we dont need this function(forgot to remove it) cause we have same thing over here https://github.com/AvineshTripathi/kubescape/blob/feature/core/pkg/registryadaptors/GCP/v1/gcpadaptor.go#L46 so I would remove this